Makefile

```diff
@@ -9,7 +9,7 @@ CLIENT_PKG_NAME := heketi-client
 SHA := $(shell git rev-parse --short HEAD)
 BRANCH := $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))
 VER := $(shell git describe --match='v[0-9].[0-9].[0-9]')
-TAG := $(shell git tag --points-at HEAD --sort=creatordate 'v[0-9].[0-9].[0-9]' | tail -n1)
+TAG := $(shell git tag --points-at HEAD 'v[0-9].[0-9].[0-9]' | tail -n1)
```

Comment:
I wanted to take the latest tag in case there are are multiple tags at HEAD.
But yeah, it's not strictly required. So if it breaks older gits - i'm fine with it. :-)